services/blockchain-app-registry/shared/metadata.js

```diff
@@ -41,16 +41,9 @@ const knownMainchainIDs = Object
 	.map(e => e.padEnd(LENGTH_CHAIN_ID, '0'));
 
 const getApplicationMetadataIndex = () => getTableInstance(
```

Comment:
Could you please rename the variable and make it a one-liner?
```suggestion
const getApplicationMetadataTable = () => getTableInstance(
```